### PR TITLE
User Guide banner fix 1: Includes main _index, /appendix, and /assets…

### PR DESCRIPTION
**Pull request by micronAMZN**
…. Removed preview banners, set zero length topics to draft, commented out links to zero length topics in this PR, other minor fixes such as removing H2s that duplicate topic title.

Signed-off-by: Mike Cronin <mikecro@amazon.com>

### CODE REVIEW

File: content/docs/user-guide/appendix/command-line-tools.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "Command line tool reference"
 date: 2021-03-02T00:23:56-05:00
```
Comment (willihay) [inline content/docs/user-guide/appendix/command-line-tools.md:3]:
```suggestion
```

File: content/docs/user-guide/appendix/cvars/_index.md [hunk 2]
```diff
@@ -2,3 +2,7 @@
 title: CVARs
 date: 2021-03-04
```
Comment (willihay) [inline content/docs/user-guide/appendix/cvars/_index.md:3]:
```suggestion
```

File: content/docs/user-guide/appendix/cvars/debugging.md [hunk 3]
```diff
@@ -3,8 +3,6 @@ description: ' Debug and fix performance issues for Open 3D Engine. '
 title: Debugging Issues
```
Comment (willihay) [inline content/docs/user-guide/appendix/cvars/debugging.md:3]:
```suggestion
title: Debug CVARs
```

File: content/docs/user-guide/appendix/file-formats.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
 ---
 title: "File format definitions"
 date: 2021-03-02T00:23:56-05:00
```
Comment (willihay) [inline content/docs/user-guide/appendix/file-formats.md:3]:
```suggestion
```

File: content/docs/user-guide/assets/fbx-settings/import/_index.md [hunk 5]
```diff
@@ -1,4 +1,5 @@
 ---
 title: Import
 date: 2021-03-04
```
Comment (willihay) [inline content/docs/user-guide/assets/fbx-settings/import/_index.md:3]:
```suggestion
```

**Review by willihay — CHANGES_REQUESTED** [review]
Let's get rid of the `date` front matter too. It messes with the order of the ToC. Can you check _all_ the topics in the appendix?

**Review by chanmosq — APPROVED** [review]

**Review by willihay — APPROVED** [review]

**Pull request merged**